Editor/CoreAPI/SettingsKeys.cs

```diff
@@ -3,22 +3,25 @@
 
 namespace AmazonGameLift.Editor
 {
-    public static class SettingsKeys
+    public enum SettingsKeys
     {
-        public const string CurrentBucketName = "CurrentBucketName";
-        public const string CurrentProfileName = "CurrentProfileName";
-        public const string CurrentRegion = "CurrentRegion";
-        public const string DeploymentScenarioIndex = "DeploymentScenarioIndex";
-        public const string DeploymentGameName = "DeploymentGameName";
-        public const string DeploymentBuildFilePath = "DeploymentBuildFilePath";
-        public const string DeploymentBuildFolderPath = "DeploymentBuildFolderPath";
-        public const string FleetName = "FleetName";
-        public const string FleetId = "FleetId";
-        public const string FleetLocation = "FleetLocation";
-        public const string ComputeName = "ComputeName";
-        public const string IpAddress = "IpAddress";
-        public const string WebSocketUrl = "WebSocketUrl";
-        public const string SelectedProfile = "SelectedProfile";
-        public const string SelectedFleetName = "SelectedFleetName";
+        CurrentBucketName,
+        CurrentProfileName,
+        CurrentRegion,
+        DeploymentScenarioIndex,
+        DeploymentGameName,
+        DeploymentBuildFilePath,
+        DeploymentBuildFolderPath,
+        FleetName,
+        FleetId,
+        FleetLocation,
+        ComputeName,
+        IpAddress,
+        WebSocketUrl,
+        SelectedProfile,
```

Comment:
How is the `CurrentProfileName` different from `SelectedProfile`? Can we just reuse the former?

Reply:
Fixed extra SettingsKey name from merge